src/panel.css

```diff
@@ -1,51 +1,248 @@
+  position: absolute;
+  display: inline-block;
+  top: 0;
+  bottom: 0;
+  margin: auto;
+  right: 15px;
+  height: 100%;
+  width: .5rem;
+  background-image: url("/img/arrow-icon-right.svg");
   background-position: center center;
+  transform: scale(1.3);
 }
 
-.img {
-  padding-right: 10px;
-  align-items: baseline;
+.highlight-on-hover span {
+  pointer-events: none;
 }
-  
-.heading {
+
+.highlight-on-hover:hover {
+  background-color: var(--grey20);
+  transition: var(--transition);
+}
+
+.fw-bottom-btn,
+.bottom-btns {
+  display: flex;
+  justify-content: space-between;
+  transition: var(--transition);
+  cursor: pointer;
+}
+
+.bottom-btns { /* Bottom "Cancel, Back, Next, Done" buttons - onboarding panels only */
+  width: 100%;
+}
+
+.bottom-btn {
+  border-bottom: 0;
+  line-height: 103%;
+  height: 3.37rem;
+  width: 50%;
+  color: var(--primaryText);
+  padding: 1.15rem 0;
   text-align: center;
-  border-bottom: 1px solid #CCC;
-  padding: 0 0 15px 0;
 }
-  
-.display {
-  display: none;
+
+.bottom-btn:first-of-type { /* Onboarding Panels only: The first bottom button ("Cancel", "Back") is always gray */
+  border-top: 1px solid rgba(24, 25, 26, 0.14);
+  border-right: 1px solid rgba(24, 25, 26, 0.14);
+  border-left: 0;
+  background-color: rgba(12,12,13, 0.05);
+  left: 0;
+}
+
+.bottom-btn:first-of-type:hover {
+  background-color: rgba(204, 204, 204, 0.5);
+}
+
+.bottom-btn:last-of-type { /* Onboarding Panels only: The second bottom button on the onboarding panels is always blue */
+  border-top: 1px solid var(--blue70);
+  border-left: 1px solid var(--blue60);
+  border-right: 0;
+  color: rgba(255, 255, 255, 1);
+  background-color: var(--blue60);
+  right: 0;
+}
+
+.bottom-btn:last-of-type:hover {
+  background-color: var(--blue70);
+}
+
+.bottom-btn,
+.bottom-btn:hover {
+  transition: var(--transition);
+}
+
+.trackers-detected-subhead::before {
+  background-image: url("/img/fenced-badge.svg");
+  background-position: center center;
+  width: 23px;
+  height: 19px;
+  display: inline-flex;
+  margin-right: 3px;
+}
+
+.img.trackers-detected {
+  background-image: url("/img/image-tracker-detected@2x(1).png");
+  height: 145px;
+  margin: 1.25rem 0 1.75rem -4px;
+  background-position: left, center;
+}
+
+.img.onboarding3 {
+  background-image: url("/img/image-example@2x.png");
+  height: 145px;
+  margin: 1rem auto 1.5rem auto;
+  background-position: center center;
+}
+
+.img {
+  display: block;
+  min-height: 100px;
+}
+
+.img,
+.highlight-on-hover::after,
+.Facebook-text::after,
+.trackers-detected-subhead::before,
+.btn-return.arrow-left {
+  content: "";
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+/* Bottom button hacks until a more reasonable solution presents itself */
+/* This eliminates (The-Powers-That-Be-Willing) a strange little white gap between
+   the bottom edge of the 'bottom-btns' and the bottom edge of the panel. */
+
+.onboarding3 .bottom-btn {
+  height: 3.41rem;
```

Comment:
I recommend rewriting this attribute to use a static `px` unit instead of `rem`.